solutions/banking/main.tf

```diff
@@ -15,51 +15,51 @@ resource "random_string" "unique_identifier" {
 data "ibm_iam_auth_token" "tokendata" {}
 
 # Resource group - create if it doesn't exist
-# module "resource_group" {
-#   source  = "terraform-ibm-modules/resource-group/ibm"
-#   version = "1.1.5"
-#   # If an existing resource group is not set (null), then create a new one
-#   resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
-#   existing_resource_group_name = var.resource_group_name
-# }
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  # If an existing resource group is not set (null), then create a new one
+  resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
+  existing_resource_group_name = var.resource_group_name
+}
 
 # create COS instance for WatsonX.AI project
-# module "cos" {
-#   source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
-#   version           = "7.5.3"
-#   resource_group_id = module.resource_group.resource_group_id
-#   cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
-#   cos_plan          = "standard"
-# }
+module "cos" {
+  source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version           = "7.5.3"
+  resource_group_id = module.resource_group.resource_group_id
+  cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
```

Comment:
Could you add a `prefix` to the name to avoid conflicts?

Reply:
I was suggesting to remove prefix variable and expose `cos_instance_name` as a variable (https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/pull/15#discussion_r1560786933)

Reply:
I'll add the prefix for now, to remain consistent with the SaaS DA.